local_infrastructure/docker-compose.yml

```diff
@@ -54,6 +54,23 @@ services:
       - ./mysql57_databases:/var/lib/mysql
       - ./my.cnf:/etc/my.cnf
 
+  mariadb103:
+    container_name: mariadb103
+    image: 'bitnami/mariadb:10.3'
+    user: root
+    restart: always
+    labels:
+      - 'traefik.enable=false'
+    network_mode: bridge
+    ports:
+      - 33103:3306
+    environment:
+      - MARIADB_ROOT_USER=root
+      - MARIADB_ROOT_PASSWORD=root
+    volumes:
+      - ./mariadb_databases:/bitnami/mariadb
```

Comment:
@Almightily Probably should be `mariadb103_databases`
Please, merge all commits in one and it will be accepted.